Dockerfile

```diff
@@ -1,9 +1,154 @@
-FROM saagie/jupyter-python-nbk:latest
+ARG PYTHON2_IMG="saagie/python:2.7.202003.76"
+ARG PYTHON3_IMG="saagie/python:3.6.202003.76"
+
+# FIXME should use a minimal image and add libs after
+ARG BASE_CONTAINER="jupyter/scipy-notebook:c7fb6660d096"
```

Comment:
One change at a time to be able to identify issues easily, so this will be addressed later on maybe by https://github.com/saagie/jupyter-python-nbk/issues/16 or in a separate commit.